extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -504,12 +507,12 @@ export class AmpLightboxViewer extends AMP.BaseElement {
 
   /**
    * Toggle description box if it has text content
-   * @param {boolean} display
    * @private
    */
-  toggleDescriptionBox_(display) {
+  showDescriptionBox_() {
```

Comment:
let's remove this and just do it inline in the `if` condition like the hide case.